Cargo.toml

```diff
@@ -218,7 +218,6 @@ test_sst_importer = { path = "components/test_sst_importer", default-features =
 test_util = { path = "components/test_util", default-features = false }
 tokio = { version = "0.2", features = ["macros", "rt-threaded", "time"] }
 zipf = "6.1.0"
-example_plugin = { path = "components/test_coprocessor_plugin/example_plugin" }
```

Comment:
Why remove this?

Reply:
This causes the test segment fault because it has a special memory allocator, and it should not be linked into tikv for it's expected to be loaded dynamically.